dao/mock_daos.go

```diff
@@ -64,7 +64,7 @@ func (src *MockSourceDao) GetById(id *int64) (*m.Source, error) {
 		}
 	}
 
-	return nil, fmt.Errorf("source not found")
+	return nil, fmt.Errorf("record not found")
```

Comment:
So - for this I would probably make the `ErrNotFound` struct contain a string variable that is the type that wasn't found. For example we'd `return nil, util.ErrNotFound{Type: "source"}` instead and make the Error() method be `return type + " not found"`